src/components/Addbox.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from "react";
+import Box from "./Box";
+class Addbox extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      lightArray: [],
+    };
+  }
+
+  moreBoxes = () => {
+    if (this.state.lightArray) {
+      this.setState({ lightArray: [...this.state.lightArray, <Box />] });
+    }
+  };
+
```

Comment:
Not too sure if you're into pseudocode but it can help ya out in the future if you're looking back.